x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/utils.ts

```diff
@@ -346,11 +419,67 @@ export async function getTestUrl(job, customUrl) {
     const docTimeFieldName = job.data_description.time_field;
 
     // Create a dummy object which contains the fields necessary to build the URL.
+    // @ts-ignore
```

Comment:
I think we can avoid using `@ts-ignore` here

Reply:
Removed in 642ce6dbe70bb719b6416e5a61184a6b966b71d7